packages/scene-composer/src/utils/nodeUtils.ts

```diff
@@ -31,7 +31,7 @@ export const findComponentByType = (
  * @returns {boolean} true if the object is a Viable Parent
  */
 const isViableParent = (object: THREE.Object3D): boolean => {
-  return !!object.userData.nodeRef;
+  return !!object.userData.targetRef;
```

Comment:
Yeah I don't agree with this design, we should revisit. Flagging models with metadata about what scene node they are makes sense because it creates a reverse mapping so we can get objects by node, or node by objects.

This is a hack that alters this behavior to solve the problem for a single feature.

Reply:
Will address.